### PR TITLE
Some improvements on transformers tool

### PR DESCRIPTION
**Pull request by wangyems**
**Description**: Describe your changes.

1. Match more patterns for Embedlayernorm fusion due to recent change on bert model
2. Fix TF benchmark GPU setting for correct inference perf
3. Force using CPU to export TF model to onnx to avoid unnecessary GPU usage
4. Add a choice to export onnx from TF model on benchmark script

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/python/tools/transformers/benchmark.py [hunk 1]
```diff
@@ -278,17 +312,21 @@ def run_tensorflow(use_gpu, model_names, model_class, precision, batch_sizes, se
...
+                logger.info("Run Tensorflow on {} with input shape {}".format(model_name,
+                                                                              [batch_size, sequence_length]))
 
                 import random
                 rng = random.Random()
                 values = [rng.randint(0, config.vocab_size - 1) for i in range(batch_size * sequence_length)]
                 input_ids = tf.constant(values, shape=(batch_size, sequence_length), dtype=tf.int32)
 
                 try:
+                    # Disable both for better inference perf
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/benchmark.py:324]: I heard that xla help performance. Do you know why they are not in your test.

**Review by tianleiwu — APPROVED** [review]

File: onnxruntime/python/tools/transformers/benchmark.py (hunk 1 above)
Comment (wangyems) [reply onnxruntime/python/tools/transformers/benchmark.py:324]: I am following the huggingface benchmark that defaults use_xla to false.
I also run tests with use_xla and find that:
cpu: ~1.1x slower
gpu: ~1.2x slower

**Review by tianleiwu — APPROVED** [review]

**Pull request merged**